public/app/plugins/datasource/prometheus/components/PromExploreQueryEditor.tsx

```diff
@@ -57,7 +57,8 @@ export const PromExploreQueryEditor: FC<Props> = (props: Props) => {
       data={data}
       ExtraFieldElement={
         <PromExploreExtraField
-          queryType={query.range && query.instant ? 'both' : query.instant ? 'instant' : 'range'}
+          // Select "both" as default option when Explore is opened
+          queryType={query.range === query.instant ? 'both' : query.instant ? 'instant' : 'range'}
```

Comment:
This `query.range === query.instant` is weird and not very intuitive. I assume false === false cannot happen, but still I think `query.range && query.instant` makes more sense when reading it.

Reply:
Added comments to explain this